packages/tpetra/core/src/Tpetra_Util.hpp

```diff
@@ -926,10 +926,12 @@ namespace Tpetra {
       Kokkos::DualView<T*, DT> x_out (label, len);
       if (leaveOnHost) {
         x_out.modify_host ();
+        // DEEP_COPY REVIEW - NOT TESTED FOR CUDA BUILD
```

Comment:
We should change a multivector communication test to use a big vector (e.g., 1M entries).  
I think this code will be exercised with a big-enough vector.